tests/models/fuyu/test_modeling_fuyu.py

```diff
@@ -246,116 +248,69 @@ def prepare_config_and_inputs_for_common(self):
+        pass
 
-"""
-    @slow
-    @require_torch_gpu
-    def test_model_8b_chat_greedy_generation_bus_color(self):
-        EXPECTED_TEXT_COMPLETION = "The bus is blue.\n|ENDOFTEXT|"
-        text_prompt_bus_color = "What color is the bus?\n"
-        model_inputs_bus_color = self.processor(text=text_prompt_bus_color, images=self.bus_image_pil)
 
-        generated_tokens = self.model.generate(**model_inputs_bus_color, max_new_tokens=10)
-        text = self.processor.tokenizer.batch_decode(generated_tokens)
-        end_sequence = text[0].split("\x04")[1]
-        clean_sequence = (
-            end_sequence[: end_sequence.find("|ENDOFTEXT|") + len("|ENDOFTEXT|")]
-            if "|ENDOFTEXT|" in end_sequence
-            else end_sequence
-        )
-        self.assertEqual(EXPECTED_TEXT_COMPLETION, clean_sequence)
+@require_torch
+@slow
+class FuyuModelIntegrationTest(unittest.TestCase):
+    def test_inference(self):
+        processor = FuyuProcessor.from_pretrained("adept/fuyu-8b")
+        model = FuyuForCausalLM.from_pretrained("adept/fuyu-8b")
 
-    @slow
-    @require_torch_gpu
-    def test_model_8b_chat_greedy_generation_chart_vqa(self):
-        # fmt: off
-        EXPECTED_TEXT_TOKENS = ["The","life expectancy","at","birth","of male","s in","","20","18","is","","80",".","7",".","\n","|ENDOFTEXT|",]
-        # fmt: on
-        expected_text_completion = " ".join(EXPECTED_TEXT_TOKENS)  # TODO make sure the end string matches
+        url = "https://huggingface.co/datasets/hf-internal-testing/fixtures-captioning/resolve/main/bus.png"
+        image = Image.open(io.BytesIO(requests.get(url).content))
 
-        text_prompt_chart_vqa = "What is the highest life expectancy at birth of male?\n"
+        text_prompt_coco_captioning = "Generate a coco-style caption.\n"
 
-        chart_image_url = (
-            "https://huggingface.co/datasets/hf-internal-testing/fixtures-captioning/resolve/main/chart.png"
-        )
-        chart_image_pil = Image.open(io.BytesIO(requests.get(chart_image_url).content))
+        inputs = processor(text=text_prompt_coco_captioning, images=image, return_tensors="pt")
+        generated_tokens = model.generate(**inputs, max_new_tokens=10)
 
-        model_inputs_chart_vqa = self.processor(text=text_prompt_chart_vqa, images=chart_image_pil)
-        generated_tokens = self.model.generate(**model_inputs_chart_vqa, max_new_tokens=10)
-        text = self.processor.tokenizer.batch_decode(generated_tokens)
+        text = processor.batch_decode(generated_tokens)
         end_sequence = text[0].split("\x04")[1]
         clean_sequence = (
             end_sequence[: end_sequence.find("|ENDOFTEXT|") + len("|ENDOFTEXT|")]
             if "|ENDOFTEXT|" in end_sequence
             else end_sequence
         )
-        self.assertEqual(expected_text_completion, clean_sequence)
 
-    @slow
-    @require_torch_gpu
-    def test_model_8b_chat_greedy_generation_bounding_box(self):
```

Comment:
Why remove the other tests? `test_model_8b_chat_greedy_generation_bus_color` and `test_model_8b_chat_greedy_generation_chart_vqa` are admittedly similar, but the bounding box one seems useful to keep particularly if we paired with with post processing to get it in `<box> ...` format